.github/workflows/ci.yaml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - diregapic-dev-branch
```

Comment:
Do we remove this now? :D

Reply:
I'm hesitant to remove it until the branch is gone.

Reply:
Fair!